ddtrace/llmobs/_integrations/anthropic.py

```diff
@@ -156,11 +164,18 @@ def _extract_output_message(self, response):
                     output_messages.append({"content": text, "role": role})
                 else:
                     if _get_attr(completion, "type", None) == "tool_use":
-                        name = _get_attr(completion, "name", "")
-                        inputs = _get_attr(completion, "input", "")
-                        output_messages.append(
-                            {"content": "[tool: {}]\n\n{}".format(name, json.dumps(inputs)), "role": role}
-                        )
+                        input_data = _get_attr(completion, "input", "")
+                        if isinstance(input_data, str):
+                            input_data = json.loads(input_data)
+                        tool_call_info = {
+                            "name": _get_attr(completion, "name", ""),
+                            "arguments": input_data,
+                            "tool_id": _get_attr(completion, "id", ""),
+                            "type": _get_attr(completion, "type", ""),
+                        }
+                        if text is None:
+                            text = ""
```

Comment:
```suggestion
```
Can we change line 162 to default to an empty string, not None, and remove this check?

Reply:
I think i saw a case where text is literally just None ("text": None) from openai output. But let me double check!

Reply:
Ahh in that case, feel free to revert, happy either way.